docs/environment-variables.md

```diff
@@ -0,0 +1,69 @@
+# Environment variables used in the MTC solution
+
+## Components shorthand
+
+* **FT** - functions throttled 
+* **FC** - functions consumption
+* **AA** - admin app
+* **PA** - pupil app (SPA)
+* **PAPI** - pupil API 
+
+## Types 
+All environment variables are strings, but they are converted to appropriate types in the component at startup.
+
+Booleans can be set to the string `'true'` or `'false'` or numeric `1` or `0` which will be converted from a string
+ to the equivalent boolean value.
+
+Env Var | Type | Default value | Required | Components | Description 
+--- | --- | --- | --- | --- | ---
+ALLOWED_WORDS | String | 'aaa,bbb,ccc,ddd,eee,dim' | Required | FC | Required to generated random school pins in the form WORD + NUMBER + NUMBER + WORD
+AZURE_SERVICE_BUS_CONNECTION_STRING | String | NULL | Required | AA, FC, FT, PAPI | Connection string as given in the Azure portal for Azure Service Bus.
```

Comment:
the following are missing...
admin app
`APPINSIGHTS_INSTRUMENTATIONKEY`
`AUTH_MODE` (local or dfe)
`DFE_SIGNON_AUTH_URL` - page user is redirected to for dfe sign on
`DFE_SIGNON_CLIENT_ID` - consumer app client id (default is mtc)
`DFE_SIGNON_CLIENT_SECRET` - auth secret
`DFE_USER_INFO_API_SECRET` - user api auth secret
`DFE_USER_INFO_API_URL` - user api url
`DFE_SIGNON_CLOCK_TOLERANCE_SECONDS` - tolerance of clock differences between consumer and service
`DFE_SIGNON_SIGNOUT_URL` - signout url
`DFE_SIGNON_SERVICES_URL` services page on dfe sign on
spa...
`TEST_PUPIL_CONNECTION_ENABLED` toggle
`TEST_PUPIL_CONNECTION_QUEUE_URL` url
`TEST_PUPIL_CONNECTION_QUEUE_TOKEN` azure storage token for endpoint
`TEST_PUPIL_CONNECTION_MAX_RETRIES` retry count

Reply:
I suggest we add the admin app env vars in on a new PR.